src/openforms/registrations/contrib/objects_api/typing.py

```diff
@@ -0,0 +1,34 @@
+from typing import Literal, TypeAlias, TypedDict
+
+from typing_extensions import Required
+
+
+class _BaseRegistrationOptions(TypedDict, total=False):
```

Comment:
Theoretically this is `total=False`, but every time the options are used, `apply_defaults_to` is used, effectively giving values for every key. So I'm wondering if it should be total?

Reply:
Making typeddicts partial/required is a pain currently, I wasn't able to make it work without redefining all the fields in a separate typeddict. It's not a big deal so I'll leave it as is for now and we could revisit once partial/required eventually lands :)